README.md

```diff
@@ -39,14 +39,63 @@ A `validate.sh` script is provided for convenience.
 
 Note: JSON Schemas only capture the NGSI simplified representation (`options=keyValues`)
 
+
```

Comment:
it looks like that in some part of the readme there where white trailing spaces, that Atom automatically remove, I will try to disable that. Even though, in general, I agree with Atom :) there should not be trailing spaces at the end of a line.